AutomatedTesting/Gem/PythonTests/Terrain/EditorScripts/Terrain_World_ConfigurationWorks.py

```diff
@@ -61,39 +62,39 @@ def Terrain_World_ConfigurationWorks():
     SET_BOX_Y_SIZE = 2048.0
     SET_BOX_Z_SIZE = 100.0
     CLAMP = 1
-
+    
     helper.init_idle()
     
-    # 1) Load the level
-    helper.open_level("", "Base")
-    helper.wait_for_condition(lambda: general.get_current_level_name() == "Base", 2.0)
-
-    # 2) Load the level components
-    terrain_world_component = hydra.add_level_component("Terrain World")
-    hydra.add_level_component("Terrain World Renderer")
-
-    # 3) Create 2 test entities, one parent at 512.0, 512.0, 50.0 and one child at the default position and add the required components
-    entity1_components_to_add = ["Axis Aligned Box Shape", "Terrain Layer Spawner", "Terrain Height Gradient List", "Terrain Physics Heightfield Collider"]
-    entity2_components_to_add = ["Vegetation Reference Shape", "Gradient Transform Modifier", "FastNoise Gradient"]
-    terrain_spawner_entity = hydra.Entity("TerrainEntity")
-    terrain_spawner_entity.create_entity(azmath.Vector3(512.0, 512.0, 50.0), entity1_components_to_add)
-    Report.result(Tests.create_terrain_spawner_entity, terrain_spawner_entity.id.IsValid())
-    height_provider_entity = hydra.Entity("HeightProviderEntity")
-    height_provider_entity.create_entity(azmath.Vector3(0.0, 0.0, 0.0), entity2_components_to_add,terrain_spawner_entity.id)
-    Report.result(Tests.create_height_provider_entity, height_provider_entity.id.IsValid())
-
-    # Give everything a chance to finish initializing.
-    general.idle_wait_frames(1)
-
-    # 4) Start the Tracer to catch any errors and warnings
+    # 1) Start the Tracer to catch any errors and warnings
     with Tracer() as section_tracer:
+        # 2) Load the level
+        helper.open_level("", "Base")
+        helper.wait_for_condition(lambda: general.get_current_level_name() == "Base", 2.0)
+        
+        # 3) Load the level components
+        terrain_world_component = hydra.add_level_component("Terrain World")
+        Report.critical_result(Tests.level_components_added, terrain_world_component is not None and hydra.add_level_component("Terrain World Renderer") is not None)
```

Comment:
consider using some new lines to make this line shorter. 
```suggestion
        Report.critical_result(
            Tests.level_components_added, 
            terrain_world_component is not None and hydra.add_level_component("Terrain World Renderer") is not None)
```
assigning the "Terrain World Renderer" to a local variable before this would also be good to do rather than executing it in the critical_result

Reply:
Both done.